src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltokentext/Example1.java

```diff
@@ -9,9 +9,13 @@
 </module>
 */
 
+package com.puppycrawl.tools.checkstyle.checks.coding.illegaltokentext;
+
 // xdoc section -- start
-public void myTest() {
+public class Example1 {
+  public void myTest() {
     String test = "a href"; // violation
     String test2 = "A href"; // OK, case is sensitive
+  }
```

Comment:
Please make it ( extra space before =)
`String test  = "a href"; // violation`

To let comments to be vertically aligned

Reply:
Not done